### ISSUE TITLE
Read multibytes language file (utf-8 encoding) in windows10

### ISSUE THREAD

**Issue by ByunghoJeon** [quantorconsulting/mkdocs_build_plantuml#10]
Hi.
I'm really useful this plugin. 

An error occurs when creating a file containing Korean characters.(UTF-8 encoded)
I explicit pass `encoding=utf-8` parameter, it works.

https://github.com/christo-ph/mkdocs_build_plantuml/blob/7f037be4ee23c8f6a65b3cf2851ed8c64322f509/mkdocs_build_plantuml_plugin/plantuml.py#L58

```python
 with open(os.path.join(diagram.directory, diagram.file), "r", encoding="utf-8") as f: 
```

thank you.

**Comment by christo-ph** [comment]
Hi, thanks for the report. Are you able to provide a merge request?

**Comment by christo-ph** [comment]
Included in 1.8.0
